Code/Tools/SceneAPI/SceneCore/Events/ExportProductList.h

```diff
@@ -53,6 +55,17 @@ namespace AZ
                 AZStd::vector<AZStd::string> m_legacyPathDependencies;
                 //! In the case of CGFs, we will have LOD export products that are dependencies of the base LOD
                 AZStd::vector<ExportProduct> m_productDependencies;
+
+                [[nodiscard]] explicit constexpr operator size_t() const
+                {
+                    size_t path_hash = AZ::IO::hash_value(AZ::IO::PathView(m_filename));
+                    return path_hash;
```

Comment:
The `path_hash` variable should be camelCase or just removed
```suggestion
                    return AZ::IO::hash_value(AZ::IO::PathView(m_filename));
```